packages/flutter_tools/lib/src/web/devfs_web.dart

```diff
@@ -317,11 +324,12 @@ class WebDevFS implements DevFS {
         mapperUrl: _filePathToUriFragment(stackTraceMapper.path),
         entrypoint: '${_filePathToUriFragment(mainPath)}.js',
       ));
-      _webAssetServer.writeFile('/main_module.js', generateMainModule(
+      _webAssetServer.writeFile('/main_module.bootstrap.js', generateMainModule(
         entrypoint: '${_filePathToUriFragment(mainPath)}.js',
       ));
       _webAssetServer.writeFile('/dart_sdk.js', dartSdk.readAsStringSync());
       _webAssetServer.writeFile('/dart_sdk.js.map', dartSdkSourcemap.readAsStringSync());
+      _webAssetServer.writeFile('/basic.digests', '{}'); // write fake digests.
```

Comment:
Just confirming. This is only necessary if the module strategy is `requireJS`. Is that the case here?

Reply:
Yeah, we're still using requireJS